corehq/ex-submodules/casexml/apps/case/util.py

```diff
@@ -21,6 +21,9 @@
 
 
 def validate_phone_datetime(datetime_string, none_ok=False, form_id=None):
+    if isinstance(datetime_string, datetime.datetime):
```

Comment:
is there just a specific workflow where this is called with a datetime and not a string? just wondering if it could be handled in the calling code rather than by making this function more ambiguous about what types it expects.